src/main/java/duke/command/CommandAdd.java

```diff
@@ -0,0 +1,52 @@
+package duke.command;
+
+import duke.task.Task;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+import java.io.IOException;
+
+/**
+ * The class that models an add command.
+ */
+public class CommandAdd extends Command {
+    Task taskToAdd;
+
+    /**
+     * Instantiates an object with given task to be added when
+     * called to execute.
+     * @param taskToAdd
+     */
```

Comment:
Consider adding a blank line between the method description and the `@param` statement to follow the Javadocs formatting in the [Java Coding Standard](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html#variables).

Reply:
Hi, thanks for pointing out these defects so I can improve my code quality. I will make modifications according to suggestions! 😄